### ISSUE TITLE
Allow to choose between vue2 and vue3

### ISSUE THREAD

**Issue by untereiner** [Kitware/trame-leaflet#13]
Is it possible to update the vue-component to be able to choose between the vue2 or vue3 leaflet wrapper ?

**Comment by jourdain** [comment]
Yes, definitely doable but if you could afford [funding that development](https://www.kitware.com/trame/), that will help to enable such feature sooner.

**Comment by untereiner** [comment]
Thank you for #15 @jourdain
